src/cljs/main/broadfcui/page/library/research_purpose.cljs

```diff
@@ -3,67 +3,131 @@
-        (this :-render-checkbox "Disease focused research" :disabled? true)
-        (this :-render-checkbox "Methods development/Validation study" :disabled? true :code :methods)
-        (this :-render-checkbox "Control set" :disabled? true :code :control)
-        (this :-render-checkbox "Aggregate analysis to understand variation in the general population" :code :aggregates)
-        (this :-render-checkbox "Study population origins or ancestry" :code :poa)
-        (this :-render-checkbox "Commercial purpose/by a commercial entity" :code :commercial)
-        #_(this :-render-checkbox "Restricted to a specific population")
-        #_(this :-render-checkbox "Other purpose" :code :other)]}])
+        (this :-render-disease-question)
+        (this :-render-checkbox "Methods development/Validation study" :methods)
+        (this :-render-checkbox "Control set" :control)
+        (this :-render-checkbox "Aggregate analysis to understand variation in the general population" :aggregates)
+        (this :-render-checkbox "Study population origins or ancestry" :poa)
+        (this :-render-checkbox "Commercial purpose/by a commercial entity" :commercial)]}])
+   :-render-disease-question
+   (fn [{:keys [state this]}]
+     [:div {:style {:position "relative"}}
+      [Checkbox {:style {:margin "0.75rem 0"}
+                 :label "Disease focused research"
+                 :checked? (:disease-checked? @state)
+                 :on-change (fn [new-val]
+                              (swap! state assoc :disease-checked? new-val))}]
+      (when (:disease-checked? @state)
+        [:div {:style {:paddingLeft "1.5rem"}}
+         (this :-render-selected-diseases)
+         [Autosuggest
+          {:url "/autocomplete/"
+           :service-prefix "/duos"
+           :caching? true
+           :inputProps {:placeholder "Search for a disease ontology value"}
+           :get-value #(.-label %)
+           :renderSuggestion (fn [suggestion]
+                               (react/create-element
+                                [:div {}
+                                 [:div {:style {:lineHeight "1.5em"}}
+                                  (.-label suggestion)
+                                  [:small {:style {:float "right"}} (.-id suggestion)]]
+                                 [:small {:style {:fontStyle "italic"}}
+                                  (.-definition suggestion)]
+                                 [:div {:style {:clear "both"}}]]))
+           :highlightFirstSuggestion false
+           :onSuggestionSelected (fn [_ suggestion]
+                                   (let [{:keys [id label]} (js->clj (.-suggestion suggestion) :keywordize-keys true)]
+                                     (swap! state update :selected-diseases assoc id label)))
+           :theme {:input {:width "100%" :marginBottom 0}
+                   :suggestionsContainerOpen {:marginTop -1 :width "100%"}}}]])])
```

Comment:
we have ontology autosuggest in three places now (counting Andrea's work in progress). should it be its own component?